worlds/ff1/Client.py

```diff
@@ -0,0 +1,333 @@
+import logging
+from collections import deque
+from copy import deepcopy
+from typing import TYPE_CHECKING, List
+
+from MultiServer import Client
```

Comment:
I think these are unused (now with my suggestion to not use typing.List)
```suggestion
from typing import TYPE_CHECKING

```